assets/src/infrastructure/ui/inputs/style.scss

```diff
@@ -1,26 +1,29 @@
 .ee-input-base-hover {
 	border-color: var(--ee-color-primary-low-contrast);
 	color: var(--ee-color-primary-high-contrast);
-	box-shadow: 0 0 0 1px var(--ee-color-primary-low-contrast);
+	box-shadow: none;
 }
 
 .ee-input-base-focus {
 	border-color: var(--ee-color-primary);
-	box-shadow: 0 0 0 1px var(--ee-color-primary);
+	box-shadow: none;
 	/* Visible in Windows high-contrast themes */
 	outline-color: transparent;
-	outline-width: 2px;
+	outline-width: 3px;
 	outline-style: solid;
 }
 
 .ee-base-inputs-styles {
 	border-color: var(--ee-border-color);
 	border-radius: var(--ee-border-radius-small);
 	border-style: solid;
-	border-width: 2px;
+	border-width: 3px;
```

Comment:
this kind of values needs to be extracted to a global variable, because it affected other components:

Reply:
![image](https://user-images.githubusercontent.com/1477453/83626931-da0e2f80-a59e-11ea-9b99-f7b0f0f951b1.png)